pkg/cli/cmds/server.go

```diff
@@ -262,6 +263,11 @@ func NewServerCommand(action func(*cli.Context) error) cli.Command {
 				EnvVar:      "K3S_CLUSTER_RESET",
 				Destination: &ServerConfig.ClusterReset,
 			},
+			cli.BoolFlag{
+				Name:        "enable-encryption",
```

Comment:
Can we just call this "secrets-encryption"

Reply:
fixed